Client/Frontend/Browser/TabManager.swift

```diff
@@ -251,12 +251,21 @@ class TabManager : NSObject {
 
 extension TabManager {
     class SavedTab: NSObject, NSCoding {
-        let url: NSURL?
+        var urls = [NSURL]()
         let isSelected: Bool
         let screenshot: UIImage?
+        let currentPage: Int
```

Comment:
You can include that `SessionData` class mentioned above directly as a variable here, too, rather than having two separate `currentPage` and `urls` vars.